pages/index.tsx

```diff
@@ -18,7 +19,9 @@ export default function Home() {
       flexDirection="column"
       width="100%"
       height="100%"
+      className="landingContainer"
     >
+      <Spacer m={2} />
```

Comment:
I still think a min-height is probably preferable over actual spacer elements from the semantic perspective and to reduce complexity; if we can't get this to work for some reason, though, we can put the spacers inside the `<Container>` to increase its intrinsic content height to prevent the top-margin gap underneath the header.